CMakeLists.txt

```diff
@@ -2,7 +2,7 @@ cmake_minimum_required(VERSION 3.12.0) # target_link_libraries with OBJECT libs
 
 project(fastfetch
     VERSION 1.7.2
-    LANGUAGES C
+    LANGUAGES C CXX # Windows part requires C++ compiler
```

Comment:
I would prefer to call `enable_language(CXX)` only on windows.

Reply:
Didn't know that